manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -532,15 +540,16 @@ func (r *FybrikApplicationReconciler) constructDataInfo(req *datapath.DataInfo,
 
 		if response, err = r.DataCatalog.GetAssetInfo(&request, credentialPath); err != nil {
 			log.Error().Err(err).Msg("failed to receive the catalog connector response")
-			return err
+			return "", err
```

Comment:
let me understand, the catalogMsg is relevant to the success response only ?

Reply:
Yes, all fields in the response are queried only if there is a successful response.